packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -49,7 +49,7 @@ export const NetworkSelectionContainer = ({
           {({ close }) => (
             <>
               {supportedNetworks?.map((chainId, i) => (
-                <div // TODO: replace with button
+                <button
                   key={chainId}
                   className="flex h-12 cursor-pointer flex-nowrap items-center justify-start space-x-3 px-12 text-lg font-light text-white hover:bg-[rgba(0,0,0,0.2)] lg:px-4 lg:text-base lg:font-normal lg:text-dark"
```

Comment:
```suggestion
                  className="flex h-12 cursor-pointer flex-nowrap items-center justify-start space-x-3 px-12 text-lg font-light text-white hover:bg-[rgba(0,0,0,0.2)] focus-visible:ring-2 focus-visible:ring-gray-4 lg:px-4 lg:text-base lg:font-normal lg:text-dark"
```

Lets add `focus-visible:ring-2 focus-visible:ring-gray-4` to the className, so that we can see the outline when tabbing through using the keyboard.